domain-models-runtime/src/main/java/org/folio/rest/persist/PreparedRowStream.java

```diff
@@ -76,7 +78,9 @@ public PreparedRowStream(PreparedStatement preparedStatement, int fetch, Tuple t
 
   @Override
   public void close() {
-    rowStream.close(close -> preparedStatement.close());
+    rowStream.close(close -> {
+      preparedStatement.close();
+    });
```

Comment:
Sonar will complain about this ("Lambdas containing only one statement should not nest this statement in a block").
https://sonarcloud.io/organizations/folio-org/rules?open=java%3AS1602&rule_key=java%3AS1602

Reply:
done.